fixtures/room_resource/delete_assigned_resource_fixtures.py

```diff
@@ -0,0 +1,32 @@
+delete_assigned_resource_mutation = '''
+    mutation {
+        deleteAssignedResource(resourceId: 2, roomId: 1) {
+            roomResource {
+                roomId
+                resourceId
+            }
+        }
+    }
+'''
+
+delete_non_existing_assigned_resource_mutation = '''
+    mutation {
+        deleteAssignedResource(resourceId: 10, roomId: 10) {
+            roomResource {
+                roomId
+                resourceId
+            }
+        }
+    }
+'''
+
+delete_assigned_resource_resp = {
```

Comment:
Kindly avoid abbreviations in variable names

Reply:
Would fix and update accordingly. 👍🏾